Source/UI/NewUI/BillingForm/Style/Cell/BillingFormTextFieldCellStyle.swift

```diff
@@ -0,0 +1,10 @@
+import UIKit
+
+// Field Cell
+public protocol BillingFormTextFieldCellStyle {
+    var backgroundColor: UIColor { get }
+    var title: InputLabelStyle? { get }
+    var hint: InputLabelStyle? { get }
+    var textfield: TextFieldStyle { get set }
+    var error: ErrorInputLabelStyle { get set }
+}
```

Comment:
Is there a way to make a field mandatory/optional? From an API point of view all of these fields are optional, Frames currently lets the merchant decide which fields to make mandatory / optional / removed.

Reply:
So all fields are optionals? can the merchant decide to remove all fields and it will show nothing?

Reply:
In theory yes. If all fields were disabled we should probably disable the billing form

Reply:
Hint and error are optionals. The merchant can decide which field should show.